source/Halibut.Tests/Halibut.Tests.csproj

```diff
@@ -10,7 +10,7 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
   <PropertyGroup Condition="!$([MSBuild]::IsOSUnixLike())">
-    <TargetFrameworks>net48;net6.0</TargetFrameworks>
+    <TargetFrameworks>net48</TargetFrameworks>
```

Comment:
Nope 😉

Reply:
hahaha, for some reason my windows wouldn't show me a run all under net48 option so I did it the manual way lol.